lib/sensors/brakes.cpp

```diff
@@ -0,0 +1,37 @@
+#include "brakes.hpp"
+
+namespace hyped::sensors {
+
+std::optional<Brakes> Brakes::create(core::ILogger &logger,
+                                     std::shared_ptr<io::IGpio> gpio,
+                                     const std::uint8_t new_pin)
+{
+  const auto reader = gpio->getReader(new_pin, io::Edge::kNone);
```

Comment:
From Tom:
getWriter right? or am I misunderstanding and this PR is for monitoring if the brakes are deployed?

Reply:
@TomLonergan03 Can you explain this? I am confused why it has to be getWriter not getReader

Reply:
What are you trying to do? Is this what deploys the brakes or is it a sensor that checks if the brakes are deployed? If it's deploying the brakes then it sends a gpio signal, so you need `getWriter`, otherwise you're right

Reply:
We're determining if the brakes are on or not based on distances. So the code is fine.